core/cmd/cli/main.go

```diff
@@ -1,3 +1,6 @@
+//go:build !unittest
```

Comment:
**question** WDYT https://peter.bourgon.org/blog/2021/04/02/dont-use-build-tags-for-integration-tests.html?

Reply:
Thanks for sharing the link! It's quite an interesting angle on the problem. IMO use of environment variables is meant to control runtime, the build [tags](https://pkg.go.dev/cmd/go#hdr-Build_constraints) is an officially recommended way to configure compiler.